flink-python/pyflink/fn_execution/table/window_aggregate_fast.pxd

```diff
@@ -25,7 +25,7 @@ cdef class NamespaceAggsHandleFunctionBase:
     cdef void retract(self, InternalRow input_data)
     cpdef void merge(self, object namespace, list accumulators)
     cpdef void set_accumulators(self, object namespace, list accumulators)
-    cdef list get_accumulators(self)
+    cpdef list get_accumulators(self)
```

Comment:
Can we add unit tests for these changes?

Reply:
I don't think this is possible...